# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mininet/uclibc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mininet/host.c.vanilla	2007-01-19 10:41:31.000000000 +0000
+++ mininet/host.c	2007-01-19 10:42:30.000000000 +0000
@@ -37,7 +37,9 @@
 #include <uint32.h>
 
 #undef dn_expand
+#if !defined (__UCLIBC__)
 #undef res_query
+#endif
 
 typedef struct { const char* name; unsigned short value; } rec;
 
